plugins/module_utils/network/nxos/rm_templates/bgp_address_family.py

```diff
@@ -0,0 +1,782 @@
+                "address_family": {
+                    '{{ nbr|d("nbr_") + afi + "_" + safi|d() + "_" + vrf|d() }}': {
+                        "maximum_paths": {
+                            "mixed": {
+                                "parallel_paths": "{{ parallel_paths }}",
+                            }
+                        }
+                    }
+                }
+            },
+        },
+        {
+            "name": "networks",
+            "getval": re.compile(
+                r"""
+                \s+network
+                \s(?P<prefix>\S+)
+                (\sroute-map\s(?P<route_map>\S+))?
+                $""",
+                re.VERBOSE,
+            ),
+            "setval": "network {{ prefix }}{{ (' route-map ' + route_map) if route_map is defined }}",
+            "result": {
+                "address_family": {
+                    '{{ nbr|d("nbr_") + afi + "_" + safi|d() + "_" + vrf|d() }}': {
+                        "networks": [
+                            {
+                                "prefix": "{{ prefix }}",
+                                "route_map": "{{ route_map }}",
+                            }
+                        ]
+                    }
+                }
+            },
+        },
+        {
+            "name": "nexthop.route_map",
+            "getval": re.compile(
+                r"""
+                \s+nexthop
+                \sroute-map\s(?P<route_map>\S+)
+                $""",
+                re.VERBOSE,
+            ),
+            "setval": "nexthop route-map {{ nexthop.route_map }}",
+            "result": {
+                "address_family": {
+                    '{{ nbr|d("nbr_") + afi + "_" + safi|d() + "_" + vrf|d() }}': {
+                        "nexthop": {
+                            "route_map": "{{ route_map }}",
+                        }
+                    }
+                }
+            },
+        },
+        {
+            "name": "nexthop.trigger_delay",
+            "getval": re.compile(
+                r"""
+                \s+nexthop
+                \strigger-delay
+                \scritical\s(?P<critical_delay>\d+)
+                \snon-critical\s(?P<non_critical_delay>\d+)
+                $""",
+                re.VERBOSE,
+            ),
+            "setval": "nexthop trigger-delay critical {{ nexthop.trigger_delay.critical_delay }} non-critical {{ nexthop.trigger_delay.non_critical_delay }}",
+            "result": {
+                "address_family": {
+                    '{{ nbr|d("nbr_") + afi + "_" + safi|d() + "_" + vrf|d() }}': {
+                        "nexthop": {
+                            "trigger_delay": {
+                                "critical_delay": "{{ critical_delay }}",
+                                "non_critical_delay": "{{ non_critical_delay }}",
+                            }
+                        }
+                    }
+                }
+            },
+        },
+        # to do
```

Comment:
Do we need something here ?

Reply:
Oh no, this one is done. Forgot to remove it. :)